tests/stub/driver_execute_query/test_driver_execute_query.py

```diff
@@ -4,7 +4,7 @@
     Driver,
     Neo4jBookmarkManagerConfig,
 )
-from tests.shared import TestkitTestCase
+from tests.shared import (TestkitTestCase, driver_feature)
```

Comment:
Please install the pre-commit hooks as described in https://github.com/neo4j-drivers/testkit/blob/5.0/CONTRIBUTING.md to get the linter to run before committing :upside_down_face: